src/ansys/fluent/core/launcher/container_launcher.py

```diff
@@ -59,9 +63,15 @@ def __init__(
         ----------
         mode : FluentMode
             Launch mode of Fluent to point to a specific session type.
+        ui : FluentUI
+            UI option of Fluent. Options are the values of the ``FluentUI`` enum.
+        graphics_driver : FluentWindowsGraphicsDriver or FluentLinuxGraphicsDriver
+            Graphics driver of Fluent. Options are the values of the
+            ``FluentWindowsGraphicsDriver`` enum in Windows or the values of the
+            ``FluentLinuxGraphicsDriver`` enum in Linux.
         product_version : str, optional
-            Select an installed version of ANSYS. The string must be in a format like
-            ``"23.2.0"`` (for 2023 R2) matching the documented version format in the
+            Installed version of Ansys. The string must be in a format like
```

Comment:
```suggestion
            Version of Ansys Fluent to launch. The string must be in a format like
```
I think specifying "installed" is a bit redundant. It has to be installed so that it can be launched locally, right? That should be expected by the user.